Gemfile

```diff
@@ -35,7 +35,7 @@ gem 'rubocop', '>= 1.0', '< 2.0'
 
 # Use Active Model has_secure_password [https://guides.rubyonrails.org/active_model_basics.html#securepassword]
 # gem "bcrypt", "~> 3.1.7"
-
+gem 'hotwire-rails'
```

Comment:
Good job setting up your app with jsbundling, however when I try to start the app using ./bin/dev, the app crashes with three errors and one of them is that `module not found for @hotwired stimulus`. I recommend running the app on your side again to very this issue and resolve it before asking for a review so that the next review can test the app

Reply:
Ok